1-JavaScript/Week4/homework/ex2-mondaysWorth.js

```diff
@@ -29,15 +29,21 @@ const mondayTasks = [
 
 const hourlyRate = 25;
 
-function computeEarnings(/* TODO parameter(s) go here */) {
+function computeEarnings(arr,hourlyRate) {
   // TODO complete this function
+  const durationTime = arr
+  .map(taskDuration => taskDuration.duration * hourlyRate)
+  .reduce((sum,taskDuration) => sum+taskDuration,0)
+  return durationTime
```

Comment:
I like a lot how you solved this exercise but the result is not correct.

Hint: Remember that the duration value represents minutes and we would like to know the earnings per hour.